project1/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <h1>The CranWife </h1>
+    <h3>
```

Comment:
Two notes here:
1. You should always keep your heading elements in sequential order. So, if you have an `h1`, the next heading should always be `h2` -- you shouldn't skip to `h3`. I expect, right now, you're doing this because of the size of the font, but we'll go over how to control this with CSS very soon.
1. You don't need a `p` inside of a heading -- it's a one or the other situation.